updatebot-core/src/main/java/io/jenkins/updatebot/kind/docker/DockerUpdater.java

```diff
@@ -78,22 +78,32 @@ protected boolean pushVersions(PushVersionChangesContext context) throws IOExcep
         return answer;
     }
 
+    private boolean checkDirectory(File dir, PushVersionChangesContext context, String name, String value) throws IOException {
+        boolean answer = false;
+		File[] files = dir.listFiles();
+		for (File file : files) {
+			if (file.isDirectory()) {
+				boolean newAnswer = checkDirectory(file, context, name, value);
+                if (!answer) {
+					answer = newAnswer;
+				}
+			} else {
```

Comment:
Some weird formatting here

Reply:
yeah though I suspect that would get sorted with the next fmt/commit - I suggest we merge, release and address this comment in another commit in the interest of fixing up the builders?